src/Explorer/Tables/QueryBuilder/QueryViewModel.tsx

```diff
@@ -111,7 +113,7 @@ export default class QueryViewModel {
     DataTableUtilities.forceRecalculateTableSize(); // Fix for 261924, forces the resize event so DataTableBindingManager will redo the calculation on table size.
   };
 
-  public ontoggleAdvancedOptionsKeyDown = (source: any, event: KeyboardEvent): boolean => {
```

Comment:
Please verify this is not breaking. If something is calling this method then the parameter order changed

Reply:
done.
https://microsoft.github.io/PowerBI-JavaScript/interfaces/_node_modules_typedoc_node_modules_typescript_lib_lib_dom_d_.keyboardevent.html